src/platform_impl/linux/menu.rs

```diff
@@ -0,0 +1,69 @@
+use std::sync::Arc;
+
+use arc_swap::ArcSwap;
+use muda::{AboutDialog, PredefinedMenuItemKind};
+
+use super::tray::Tray;
+
+pub fn muda_to_ksni_menu_item(
+    item: Arc<ArcSwap<muda::CompatMenuItem>>,
+) -> ksni::menu::MenuItem<Tray> {
+    match &**item.load() {
+        muda::CompatMenuItem::Standard(menu_item) => {
+            let id = menu_item.id.clone();
+            match &menu_item.predefined_menu_item_kind {
+                Some(PredefinedMenuItemKind::About(Some(metadata))) => {
+                    let about_dialog = AboutDialog::new(metadata.clone());
+                    ksni::menu::StandardItem {
+                        label: menu_item.label.clone(),
+                        enabled: menu_item.enabled,
+                        icon_data: menu_item.icon.clone().unwrap_or_default(),
+                        activate: Box::new(move |_| {
+                            about_dialog.show();
+                        }),
+                        ..Default::default()
+                    }
+                    .into()
+                }
+                _ => ksni::menu::StandardItem {
+                    label: menu_item.label.clone(),
+                    enabled: menu_item.enabled,
+                    icon_data: menu_item.icon.clone().unwrap_or_default(),
+                    activate: Box::new(move |_| send_menu_event(&id)),
```

Comment:
I think this match branch includes `PredefinedMenuItem` other than `PredefinedMenuItem::about` but `PredefinedMenuItem` shouldn't send events to users, it should be handled internally

Reply:
Any predefined item other than `about` doesn't make much sense for tray icon menus, so they shouldn't be added in the first place. I wouldn't know how to handle these events.

Btw. that would mean, we can remove the `libxdo` feature from `tray-icon`, right?

Reply:
I guess we can, it will still be needed for `muda`

Reply:
I have removed the libxdo feature now.